grafana-plugin/src/models/alertgroup/alertgroup.helpers.ts

```diff
@@ -33,6 +35,26 @@ export class AlertGroupHelper {
     return (await onCallApi().POST('/alertgroups/bulk_action/', { params: {}, body: data })).data;
   }
 
+  @AutoLoadingState(ActionKey.INCIDENTS_BULK_UPDATE)
+  static async updateBulkActionAndRefresh(
+    data: ApiSchemas['AlertGroupBulkActionRequest'],
+    alertGroupStore: AlertGroupStore,
+    onFinally?: () => void
```

Comment:
This made more sense to be here rather than in the component, it also accepts `AutoLoadingState`